cypress/e2e/smoke/pending_actions.cy.js

```diff
@@ -0,0 +1,63 @@
+const SAFE = 'gor:0xCD4FddB8FfA90012DFE11eD4bf258861204FeEAE'
+
+describe('Pending actions', () => {
+  it('should navigate to the form', () => {
+    cy.connectE2EWallet()
+
+    cy.visit('/welcome')
+
+    // Close cookie banner
+    cy.contains('button', 'Accept all').click()
+
+    // Ensure wallet is connected to correct chain via header
+    cy.contains('E2E Wallet @ Görli')
+  })
+
+  it('should add the Safe with the pending actions', () => {
+    // Enters Loading Safe form
+    cy.contains('button', 'Add existing Safe').click()
+    cy.contains('Connect wallet & select network')
+
+    // Inputs the Safe address
+    cy.get('input[name="address"]').type(SAFE)
+    cy.contains('Next').click()
+
+    cy.contains('Owners and confirmations')
+    cy.contains('Next').click()
+
+    cy.contains('Add').click()
+  })
+
+  it('should display the pending actions in the Safe list sidebar', () => {
+    cy.get('aside').within(() => {
+      cy.get('[data-testid=ChevronRightIcon]').click({ force: true })
+    })
+
+    cy.get('li').within(() => {
+      cy.contains('0xCD4F...eEAE').should('exist')
+
+      cy.get('img[alt="E2E Wallet logo"]').next().contains('3').should('exist')
+      cy.get('[data-testid=CheckIcon]').next().contains('3').should('exist')
+
+      // click on the pending actions
+      cy.get('[data-testid=CheckIcon]').next().click()
+    })
+  })
+
+  it('should have the right number of queued and signable transactions', () => {
+    // Navigates to the tx queue
+    cy.contains('h3', 'Transactions').should('be.visible')
+
+    // contains 3 queued transactions
+    cy.get('span:contains("out of 2")').should('have.length', 3)
+
+    // TODO: This is a workaround to ensure the wallet is connected after page navigation
+    window.localStorage.setItem('SAFE_v2__lastWallet', JSON.stringify('E2E Wallet'))
```

Comment:
The command we have (`connectE2EWallet`) was not taking effect after the page navigation and I didn't find a better way than setting the localStorage manually.
Any hints or help is very welcome.